spec/lib/state_machine_spec.rb

```diff
@@ -44,10 +44,10 @@
     expect(test.second_state_at).not_to be_nil
 
     expect(test.has_been_made_second_state?).to be true
-    expect{ test.has_been_made_fourth_state? }.to raise_error(NotImplementedError)
+    expect{ test.has_been_made_fifth_state? }.to raise_error(NotImplementedError)
```

Comment:
why this change?

Reply:
I added fourth_state_at to the model so that i could assert it wasn't getting set when the callback fails in the other test. That then caused this to not raise because having the _at column is what is required to generate the has_been_made function